cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -153,7 +156,7 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
 
         ret.iterable = frontend.expressionHandler.handle(node.iter)
         ret.variable = frontend.expressionHandler.handle(node.target)
-        ret.statement = makeBlock(node.body).codeAndLocationFromChildren(node)
+        ret.statement = makeBlock(node.body).codeAndLocationFromChildren(node as Python.ASTBASEstmt)
```

Comment:
Can we get rid of this cast by making NormalOrAsyncFor depend on ASTBASEStmt? Do we want this??? I don't like this cast.

Reply:
We can't because one is an interface and one is an abstract class. That's the drawback of having it this way and not extend ASTFunctionDef on NormalOrAsyncFor instead of ASTBASEStmt

Reply:
:+1: